src/app/containers/SocialEmbed/index.jsx

```diff
@@ -29,7 +30,10 @@ const SocialEmbedContainer = ({ blocks, source }) => {
 
   const { model } = blocks[0];
   const oEmbed = path(['blocks', 0, 'model', 'oembed'], model);
-  const oEmbedPosition = path(['indexOfType'], oEmbed) + 1;
+  const oEmbedIndexOfType = path(['indexOfType'], oEmbed);
+  const oEmbedPosition = is(Number, oEmbedIndexOfType)
+    ? oEmbedIndexOfType + 1
+    : undefined;
```

Comment:
maybe this would work

```suggestion
  const oEmbedPosition = is(Number, oEmbedIndexOfType) && oEmbedIndexOfType + 1;
```